lib/peeps.rb

```diff
@@ -0,0 +1,40 @@
+require 'pg'
+
+class Peeps
+
+  attr_reader :id, :message, :date
+
+  def initialize(id:, message:, date:)
+    @id = id
+    @message = message
+    @date = date
+  end
+
+def self.all
+
+  if ENV['ENVIRONMENT'] == 'test'
+    con = PG.connect(dbname: 'chitter_test')
+  else
+    con = PG.connect(dbname: 'chitter')
+  end
+
+  # time = con.exec("SELECT TO_CHAR(NOW() :: DATE, 'dd/mm/yyyy');")
+  result = con.exec("SELECT * FROM peeps;")
+  result.map {|peep| Peeps.new(id: peep['id'], message: peep['message'], date: peep['posting_date'])}
+end
+
+
+def self.create(message:)
+
+  if ENV['ENVIRONMENT'] == 'test'
+    con = PG.connect(dbname: 'chitter_test')
+  else
+    con = PG.connect(dbname: 'chitter')
+  end
+
```

Comment:
You could use `Time.new` which can give you the date & time - then that can be saved into a variable `time` which is then inserted into the DB along with the message.